guides/common/modules/proc_configuring-your-deployment-to-run-ansible-roles.adoc

```diff
@@ -1,21 +1,12 @@
 [id="configuring-your-deployment-to-run-Ansible-roles_{context}"]
-
 = Configuring your Deployment to Run Ansible Roles
```

Comment:
What does "your deployment" mean? Can I replace this with "your {Project}"?

Reply:
I believe so.